tcf_website/static/search/search.css

```diff
@@ -0,0 +1,8 @@
+hr {
```

Comment:
maybe something like `.search hr` so as to not accidentally style other `hr`s?

Reply:
fixed